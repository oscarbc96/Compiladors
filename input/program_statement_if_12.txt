calc off

if (not true) then 
  123
  123
  123
fi
