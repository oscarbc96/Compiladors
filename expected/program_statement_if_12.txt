1: GOTO 8
2: PARAM 123
3: CALL PUTI,1
4: PARAM 123
5: CALL PUTI,1
6: PARAM 123
7: CALL PUTI,1
8: HALT
